bsp/boards/scum/board_info.h

```diff
@@ -30,6 +30,23 @@ to return the board's description.
 #define SCHEDULER_WAKEUP()                  
 #define SCHEDULER_ENABLE_INTERRUPT()        
 
+// ==== SCuM RF timer specific 
+
+// since SCuM uses 500KHz, but the statck protocol is designed for 32KHz.
+// the following two marco are used to convert the couter value bewteen
+// two different frequency clocks. 500000/32768 is approximately 61/4.
+#define TIMER_COUTER_CONVERT_32K_TO_500K(value)    value*61/4

```

Comment:
I'm a bit afraid of these several operations in one line. We can try, but could you please add parenthesis so it's clear in what order you want the division and multiplication to happen?